bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -56,11 +58,11 @@
 
     // Note -- synchronized though I guess this should always be called from
     // gui thread...
-    public static synchronized String formatBrightness(Magnitude mag) {
+    private static synchronized String formatBrightness(Magnitude mag) {
```

Comment:
Why should it be called from the the GUI thread? It looks like no swing calls are done here, are they?